pages/options/index.js

```diff
@@ -28,12 +28,18 @@ class OptionsPage extends Component {
         A: false,
         S: false,
       },
-      gender: '',
+      gender: 'All',
+      showHours: false,
+      sortByDistance: false,
+      showDropDown: false,
+      genders: ['All', 'Men', 'Women'],
```

Comment:
is this used for display purposes, or just state tracking? If its the latter then then we should stick to the one letter codes.  Also I'm not sure how much of this gets in @thebritican's way or will conflict with his work.